File: Estructura_de_datos_C++/Busqueda_Lineal.cpp
#include <iostream>
int main(){
    int numeros [4] = {10, 20, 30, 40};
    int encontrar_Numero;
    std::cout << "Ingrese un numero para buscar:";
    std::cin >> encontrar_Numero;
    bool encontrado = false;
    for (int i = 0; i < 4; i++){
        if (numeros[i] == encontrar_Numero){
            encontrado = true;
            std::cout << "Numero encontrado en la posicion: " <<i << std ::endl;
            
            break;
        
        }
        
    }
    if (!encontrado){
            std::cout << "Numero no encontrado" << std::endl;
        }

    return 0;

}

File: Arreglos_Python/Busqueda_Lineal.py
numeros = [40,80,90,20,10]
# Pedir un numero para buscar
numero_Buscar = int(input("ingrese un numero a buscar: "))
encontrado = False
# Recorrer la lista para buscar el numero
for i in range(len(numeros)):
    if numeros[i] == numero_Buscar:
        print("Número encontrado en la posición:", i)
        encontrado = True
        break
    
# Si no se encontró el número
if not encontrado: 
    print("Número no encontrado en la lista.")
